system/datatypes/peer_reviewed_article.php

```diff
@@ -602,7 +618,7 @@ private function updateDiscussionsContainer($type, $discussionsContainer) {
 			}
 
 			// display comment form if the discussion is still open
-			if ($hasComments && !$closed && self::STATUS_PUBLISHED !== $status) {
+			if ($hasComments && !$closed) {
```

Comment:
I agree with this change, it was implemented like this on request.. Perhaps a discussion is needed to see what is the proper implementation.